packages/augur-state/src/db/DB.test.ts

```diff
@@ -1,39 +1,134 @@
-import {DB} from "./DB";
-import {ContractDependenciesEthers} from "contract-dependencies-ethers";
-import {Augur} from "@augurproject/api";
-import {uploadBlockNumbers} from "@augurproject/artifacts";
-import settings from "@augurproject/state/src/settings.json";
-import {makeMock} from "../utils/MakeMock";
-import {JsonRpcProvider} from "ethers/providers/json-rpc-provider";
-import {EthersProvider} from "ethers-provider";
+import { TrackedUsers } from "./TrackedUsers";
+import { DB } from "./DB";
+import { EthersProvider } from "ethers-provider";
+import { ContractDependenciesEthers } from "contract-dependencies-ethers";
+import { Augur, UserSpecificEvent } from "@augurproject/api";
+import { uploadBlockNumbers } from "@augurproject/artifacts";
+
+const settings = require("../settings.json");
+import { makeMock } from "../utils/MakeMock";
+import { IBlockAndLogStreamerListener } from "./BlockAndLogStreamerListener";
+import { JsonRpcProvider } from "ethers/providers";
 
 const mock = makeMock();
 const TEST_NETWORK_ID = 4;
 const defaultStartSyncBlockNumber = uploadBlockNumbers[TEST_NETWORK_ID];
+const genericEventNames: Array<string> = [
+  "DisputeCrowdsourcerCompleted",
+  "DisputeCrowdsourcerCreated",
+  "DisputeWindowCreated",
+  "MarketCreated",
+  "MarketFinalized",
+  "MarketMigrated",
+  "MarketParticipantsDisavowed",
+  "ReportingParticipantDisavowed",
+  "TimestampSet",
+  "UniverseCreated",
+  "UniverseForked"
+];
+const userSpecificEvents: Array<UserSpecificEvent> = [
+  {
+    "name": "CompleteSetsPurchased",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 2
+  },
+  {
+    "name": "CompleteSetsSold",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 2
+  },
+  {
+    "name": "DisputeCrowdsourcerContribution",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 1
+  },
+  {
+    "name": "DisputeCrowdsourcerRedeemed",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 1
+  },
+  {
+    "name": "InitialReporterRedeemed",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 1
+  },
+  {
+    "name": "InitialReportSubmitted",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 1
+  },
+  {
+    "name": "InitialReporterTransferred",
+    "numAdditionalTopics": 2,
+    "userTopicIndex": 2
+  },
+  {
+    "name": "MarketMailboxTransferred",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 2
+  },
+  {
+    "name": "MarketTransferred",
+    "numAdditionalTopics": 2,
+    "userTopicIndex": 1
+  },
+  {
+    "name": "OrderCanceled",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 2
+  },
+  {
+    "name": "OrderCreated",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 0
+  },
+  {
+    "name": "OrderFilled",
+    "numAdditionalTopics": 2,
+    "userTopicIndex": 1
+  },
+  {
+    "name": "TokensTransferred",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 2
+  },
+  {
+    "name": "TradingProceedsClaimed",
+    "numAdditionalTopics": 3,
+    "userTopicIndex": 2
+  }
+];
```

Comment:
`genericEventNames` & `userSpecificEvents` got moved to Augur.ts in master (and most of the events in `userSpecificEvents` got moved over to `genericEventNames`). These changes should be made to this branch so we don't undo those changes.